Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift

```diff
@@ -90,7 +90,7 @@ extension AbsolutePath {
         var ancestorPath = AbsolutePath("/")
         for component in components.dropFirst() {
             let nextPath = ancestorPath.appending(component: component)
-            if path.contains(nextPath) {
+            if path.isDescendantOfOrEqual(to: nextPath) {
```

Comment:
What this is about? 🤔

Reply:
just fixing warnings as suggested by Xcode